recipes/pole/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "0.5":   
+    url: "https://github.com/luc-c/Pole/archive/refs/tags/0.5.zip"
```

Comment:
I know this is your fork, but I'd like to ask what's the difference with the original https://github.com/otofoto/Pole and why one would be chosen above there other, thanks!